### ISSUE TITLE
bug(scope): Is this dead ?

### ISSUE THREAD

**Issue by Kaneki010** [Not-Quite-RARBG/main#60]
### Current Behavior

Nothing

### Expected Behavior

Updates 

### Steps To Reproduce

View github

### Environment

```markdown
- **OS**:
- **Browser**:
```

**Comment by Zabooby** [comment]
I had a felling it wouldn't last long, but was surprised to see how fast updates stopped. 

Seems like they took donations and disappeared.

**Comment by AnonymousWP** [comment]
> I had a felling it wouldn't last long, but was surprised to see how fast updates stopped.
> 
> Seems like they took donations and disappeared.

Those are quite the amount of assumptions you're making here. If you wonder why the project seems dead, then just ask instead of just making assumptions, such as thinking we "just took donations" and disappeared. The server costs have been paid by the donations, nothing else; nobody of our team got/gets paid for this. Anyway:

Read this: https://github.com/Not-Quite-RARBG/main/issues/50#issuecomment-1657278973

**Comment by Zabooby** [comment]
> Those are quite the amount of assumptions you're making here. If you wonder why the project seems dead, then just ask instead of just making assumptions, such as thinking we "just took donations" and disappeared. The server costs have been paid by the donations, nothing else; nobody of our team got/gets paid for this. Anyway, read this: [#50 (comment)](https://github.com/Not-Quite-RARBG/main/issues/50#issuecomment-1657278973)

Sure I could have asked, but the burden is on you guys to provide us with continuous updates instead of going silent for so long. People have donated money and to my knowledge this is the first time you are explaining what you actually did with their money. On top of that this project already had a rocky start, which would make people doubtful already, not to mention that there is still currently no functioning product. What I mentioned in my previous comment has happened many times before. 

Going forward it would be great to see your team release a clear and detailed roadmap for the project, providing us with regular updates. I really want to see this website to succeed and being clearer with users would be a great start. :)

**Comment by Kaneki010** [comment]
> > Those are quite the amount of assumptions you're making here. If you wonder why the project seems dead, then just ask instead of just making assumptions, such as thinking we "just took donations" and disappeared. The server costs have been paid by the donations, nothing else; nobody of our team got/gets paid for this. Anyway, read this: [#50 (comment)](https://github.com/Not-Quite-RARBG/main/issues/50#issuecomment-1657278973)
> 
> Sure I could have asked, but the burden is on you guys to provide us with continuous updates instead of going silent for so long. People have donated money and to my knowledge this is the first time you are explaining what you actually did with their money. On top of that this project already had a rocky start, which would make people doubtful already, not to mention that there is still no functioning product. What I mentioned in my previous comment has happened many times before.
> 
> Going forward it would be great to see your team release a clear and detailed roadmap for the project, providing us with regular updates. I really want to see this website succeed and being clearer with users would be a great start. :)

damn , i was probably the only guy during the whole og dev controversy who thought this would develop into a serious thing
but i was wrong sadly , the project seems dead :(
damn im still not used to life without rarbg

**Comment by AnonymousWP** [comment]
> > Those are quite the amount of assumptions you're making here. If you wonder why the project seems dead, then just ask instead of just making assumptions, such as thinking we "just took donations" and disappeared. The server costs have been paid by the donations, nothing else; nobody of our team got/gets paid for this. Anyway, read this: [#50 (comment)](https://github.com/Not-Quite-RARBG/main/issues/50#issuecomment-1657278973)
> 
> Sure I could have asked, but the burden is on you guys to provide us with continuous updates instead of going silent for so long. People have donated money and to my knowledge this is the first time you are explaining what you actually did with their money. On top of that this project already had a rocky start, which would make people doubtful already, not to mention that there is still no functioning product. What I mentioned in my previous comment has happened many times before.
> 
> Going forward it would be great to see your team release a clear and detailed roadmap for the project, providing us with regular updates. I really want to see this website succeed and being clearer with users would be a great start. :)

Yes, I completely agree, but I'm pretty sure another member has publicly told what we're going to do with the donations. I'm afraid a roadmap is going to be hard, as I'm the only member left that's communicative reachable (to a certain extent). Here some updates (CC: @Kaneki010):

![image](https://github.com/Not-Quite-RARBG/main/assets/50231698/f6be42a6-8a6b-4549-9422-ba49b607d483)

![image](https://github.com/Not-Quite-RARBG/main/assets/50231698/31a9f163-3b99-432e-a1bb-19857fa16703)

![image](https://github.com/Not-Quite-RARBG/main/assets/50231698/c68db2cd-c3cd-46ae-8b7d-658aca92bb9c)
